gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/source/BackfillHiveSource.java

```diff
@@ -74,4 +74,15 @@ public boolean shouldCreateWorkunit(Partition sourcePartition, LongWatermark low
     // If no whitelist is set, all partitions of a dataset are backfilled
     return true;
   }
+
+  @Override
+  public boolean isOlderThanLookback(Partition partition) {
```

Comment:
As discussed offline, when specific partitions are whitelisted no discovery / partition filtering needs to happen since we already know what partitions to process. Hence ignoring lookback makes sense. Thanks for the clarification.